pontoon/api/tests/test_schema.py

```diff
@@ -0,0 +1,149 @@
+from django.test import TestCase
+from django.utils.encoding import force_text
+
+
+class TestCyclicQueries(TestCase):
+
+    def test_projects(self):
+        body = {
+            'query': '''{
+                projects {
+                    name
+                }
+            }'''
+        }
+
+        response = self.client.get('/graphql', body,
+                                   HTTP_ACCEPT="application/json")
+
+        self.assertEqual(response.status_code, 200)
+        self.assertJSONEqual(
+            force_text(response.content),
+            {"data": {"projects": [
+                {"name": "Pontoon Intro"}]}})
```

Comment:
Please put closing brackets and parentheses on the same level as the opening one. Like this:

```python
self.assertJSONEqual(
    force_text(response.content),
    {
        "data": {
            "projects": [
                {"name": "Pontoon Intro"}
            ]
        }
    }
)
```

It takes a lot more lines but I think it is more readable and easier to maintain. One thing you can ask yourself when writing code is: if someone changes that, will the diff be readable? That's why we add commas at the end of lines in lists even if it's the last element, for example. 

This:

```diff
  [
      'abc',
+     'def',
  ]
```

... is better than this:

```diff
  [
-     'abc'
+     'abc',
+     'def',
  ]
```